File: ProgramTwo/code.py
def factorial(n):
    result = 1
    
    if n == 0 or n == 1:
        return result
    
    for i in range(2,n+1):
        result *= i
        
    return result

print(factorial(0))
print(factorial(1))
print(factorial(4))
print(factorial(5))

File: ProgramTwo/code.js
function factorial(n) {
    let result = 1

    if(n === 0 || n === 1) {
        return result
    }

    for(let i = n; i >=  2; i--){
        result = result * i
    }

    return result
}

console.log(factorial(0));
console.log(factorial(1));
console.log(factorial(4));
console.log(factorial(5));
